atomic_reactor/util.py

```diff
@@ -785,3 +787,71 @@ def get_config_from_registry(image, registry, digest, insecure=False,
     logger.debug('Image %s:%s has config:\n%s', context, tag, blob_config)
 
     return blob_config
+
+def df_parser(df_path, workflow=None, cache_content=False, env_replace=True, parent_env=None):
```

Comment:
This function gives a lot of flexibility. But most of it is not yet used. Consider having a wrapper function for the most common use case which can be done by simply taking the workflow object.